chd_test.go

```diff
@@ -50,7 +50,7 @@ func TestCHDBuilder(t *testing.T) {
 	}
 	c, err := b.Build()
 	assert.NoError(t, err)
-	assert.Equal(t, 7, len(c.keys))
+	assert.Equal(t, 15, len(c.keys))
```

Comment:
**"A perfect hash function maps a static set of n keys into a set of m integer numbers without collisions, where m is greater than or equal to n. If m is equal to n, the function is called minimal."**

Therefore, by definition, the number of entries in the resulting hash table must be equal to the number of input keys.